src/containers/WaGroups/GroupDetails.tsx/GroupDetails.tsx

```diff
@@ -137,13 +151,17 @@ export const GroupDetails = () => {
     );
   }
 
+  if (groupDataLoading) {
+    return <CircularProgress data-testid="loading" />;
```

Comment:
I think we have a loading component please reuse that in all places if possible.  We have configured that to be center aligned on the screen. This adds a loading at the top left of the screen which looks weird